cli/src/stake.rs

```diff
@@ -2477,7 +2475,7 @@ pub fn get_epoch_boundary_timestamps(
                 break block_time;
             }
             Err(_) => {
-                epoch_start_slot += 1;
+                epoch_start_slot = epoch_start_slot.saturating_add(1);
```

Comment:
Except for a test case that is checking for a boundary case, it would take a practically infinite amount of time to saturate a `u64` %)
But, I've changed this to a checked add.
This whole addition is actually incorrect in the first place.
Added a note.